index.js

```diff
@@ -86,6 +86,8 @@ class Applesign {
       if (this.config.withoutWatchapp) {
         tasks.push(this.removeWatchApp());
         tasks.push(this.removePlugins());
+      } else if (this.config.xctestsOnly) {
```

Comment:
This boolean logic is starting to be tricky. So, if i understand it correctly -x does -w and the inverse of -W. This is:

* -w : remove watchapps and plugins
* -W : remove only the xctests
* -x : only works when -w is not specified, and it removes the plugins, not the watchapp is not removed

From what I understand with your patch is that removePlugins() is also removing the xctests, if so, i think it will be better to make removePlugins() not remove the xctests at all, and just call removeXCTests() to do so.

Also i think its cleaner to skip them instead of renaming to avoid the purge below

Reply:
So after this we can keep the internal configuration options, but change the flags to something like:

* -w : remove watchapps
* -W : remove plugins
* -x : remove xctests

how does it sound to you?

Reply:
That sounds good to me!  I can make the changes and update the PR.  Should -w still remove plugins in addition to the watchapps?  I think it would be clearer if it did not (or the docs should be updated to note that the plugins are removed).

Reply:
I went ahead and pushed changes:
- -w only removes watchapps
- -W removes plugins but not xctests
- -x removes xctests

Let me know if you'd like other changes!